scene/gui/tree.cpp

```diff
@@ -134,6 +135,14 @@ TreeItem::TreeCellMode TreeItem::get_cell_mode(int p_column) const {
 	return cells[p_column].mode;
 }
 
+void Tree::set_dragExited(bool b){
```

Comment:
As you can see, all methods of the Godot API use snake_case, so you should do the same.